### PR TITLE
Remove reference to ability to use port/username

### PR DESCRIPTION
**Pull request by Tylarb**
Port and username options do not work in current gpssh release.

### CODE REVIEW

**Comment by Tylarb** [comment]
Demonstrating the issue which the logs indicate should work:


~~~
[gpadmin@master ~]$ cat hostfile
node1
node2

[gpadmin@master ~]$ gpssh -f hostfile
=> hostname
[node1] node1
[node2] node2
=> exit

[gpadmin@master ~]$ cat hostfile_port
node1:22
node2:22
[gpadmin@master ~]$ gpssh -f hostfile_port
Traceback (most recent call last):
  File "/usr/local/greenplum-db/./bin/gpssh", line 288, in <module>
    main()
  File "/usr/local/greenplum-db/./bin/gpssh", line 247, in main
    hostlist.parseFile(GV.opt['-f'])
  File "/usr/local/greenplum-db/lib/python/gppylib/util/ssh_utils.py", line 91, in parseFile
    self.add(line, i + 1)
  File "/usr/local/greenplum-db/lib/python/gppylib/util/ssh_utils.py", line 70, in add
    raise HostNameError(str(e), lineno)
gppylib.util.ssh_utils.HostNameError: illegal IP address string passed to inet_pton at line 1


[gpadmin@master ~]$ cat hostfile_username
gpadmin@node1
gpadmin@node2
[gpadmin@master ~]$ gpssh -f hostfile_username
Traceback (most recent call last):
  File "/usr/local/greenplum-db/./bin/gpssh", line 288, in <module>
    main()
  File "/usr/local/greenplum-db/./bin/gpssh", line 247, in main
    hostlist.parseFile(GV.opt['-f'])
  File "/usr/local/greenplum-db/lib/python/gppylib/util/ssh_utils.py", line 91, in parseFile
    self.add(line, i + 1)
  File "/usr/local/greenplum-db/lib/python/gppylib/util/ssh_utils.py", line 63, in add
    raise HostNameError(host, lineno)
gppylib.util.ssh_utils.HostNameError: gpadmin@node1 at line 1
~~~


The "@" syntax and ipv6 are blocked as per these checks: 
https://github.com/greenplum-db/gpdb/blob/master/gpMgmt/bin/gppylib/util/ssh_utils.py#L58

Discussed with @doty-pivotal  - these are unlikely to be fixed, so the documentation should be adjusted in the interim.

**Comment by mkiyama** [comment]
@doty-pivotal 
Should this fix be backported to the 5.x and 4.3.x releases.

File: gpdb-doc/dita/utility_guide/admin_utilities/gpssh.xml [hunk 1]
```diff
@@ -82,11 +82,10 @@
                 <plentry>
                     <pt>-f <varname>hostfile_gpssh</varname></pt>
                     <pd>Specifies the name of a file that contains a list of hosts that will
-                        participate in this SSH session. The host name is required, and you can
-                        optionally specify an alternate user name and/or SSH port number per host.
-                        The syntax of the host file is one host per line as follows:</pd>
+                        participate in this SSH session.The syntax of the host file is one host
+                        per line as follows:</pd>
                     <pd>
```
Comment (mkiyama) [inline gpdb-doc/dita/utility_guide/admin_utilities/gpssh.xml]: If the optional elements of the syntax is no longer required, I would remove the \<pd> ... \</pd> that contains the example and just say
The syntax of the host file is one host per line. ~~as follows:~~

**Comment by Tylarb** [comment]
@mkiyama  Adjusted commit per your suggestion. I know the issue is present in all 5.x release. 

The code indicates that the above issue (i.e. documentation is incorrect) in all 4.3.x releases also, so it should be fixed there as well. I can submit a separate PR for the 4.3 docs if you like: 

~~~
util(f413ff3b)$ git status ssh_utils.py
HEAD detached at 4.3.10.0
nothing to commit, working tree clean


 50     def add(self, host, lineno=0):
 51         '''Add a host to the hostlist.'''
 52
 53         # we don't allow the user@ syntax here
 54         if host.find('@') >= 0:
 55             raise HostNameError(host, lineno)
 56
 57         # MPP-13617 - check for ipv6
 58         if host.find(':') >= 0:
 59             try:
 60                 socket.inet_pton(socket.AF_INET6, host)
 61             except socket.error, e:
 62                 raise HostNameError(str(e), lineno)
 63
 64         # MPP-13617 - check for ipv4
 65         if host.find('.') >= 0:
 66             octs = host.split('.')
 67             if len(octs) == 4 and False not in [o.isdigit() for o in octs]:
 68                 try:
 69                     socket.inet_pton(socket.AF_INET, host)
 70                 except socket.error, e:
 71                     raise HostNameError(str(e), lineno)
~~~

**Comment by mkiyama** [comment]
@Tylarb 
I created a tracker story to update the 5.x and 4.3.x docs.

thanks

File: gpdb-doc/dita/utility_guide/admin_utilities/gpssh.xml [hunk 2]
```diff
@@ -82,12 +82,8 @@
                 <plentry>
                     <pt>-f <varname>hostfile_gpssh</varname></pt>
                     <pd>Specifies the name of a file that contains a list of hosts that will
-                        participate in this SSH session. The host name is required, and you can
-                        optionally specify an alternate user name and/or SSH port number per host.
-                        The syntax of the host file is one host per line as follows:</pd>
-                    <pd>
-                        <codeblock>[<varname>username</varname>@]<varname>hostname</varname>[:<varname>ssh_port</varname>]</codeblock>
-                    </pd>
+                        participate in this SSH session.The syntax of the host file is one host
```
Comment (dyozie) [inline gpdb-doc/dita/utility_guide/admin_utilities/gpssh.xml]: Need a space after the period here.  Otherwise, looks good!

**Pull request merged**